nl.emphisia.icon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "nl.emphisia.icon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "runtime-version" : "46",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "folder_icon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "folder_icon=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "folder_icon",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://github.com/youpie/Iconic/releases/latest/download/Iconic-Release.tar.xz",
```

Comment:
this should be a tagged release.

Reply:
@hfiguiere what exactly do you mean by that?